src/treeview/node.js

```diff
@@ -129,6 +136,20 @@ export default class Node {
 	}
 
 	/**
+	 * Clones this node.
+	 *
+	 * @method treeView.Node#clone
+	 * @returns {treeView.Node} Clone of this node.
+	 */
+
+	/**
+	 * Checks if provided node is the same as this node.
```

Comment:
They are not the same. Elements can have different children. You could note it here to clear what does to it mean that nodes are similar.